RELEASE.md

```diff
@@ -0,0 +1,81 @@
+Release Instructions
+====================
+
+Set up Sonatype Account
+-----------------------
+* Sign up for a Sonatype JIRA account [here](https://issues.sonatype.org)
+* Click *Sign Up* in the login box, follow instructions
+
+Get access to repository
+------------------------
+* Go to [community support](https://issues.sonatype.org/browse/OSSRH)
+* Ask for publish rights by creating an issue similar to [this one](https://issues.sonatype.org/browse/OSSRH-16798)
+
+Set up PGP keys
+---------------
+* Generate the key ```gpg --gen-key```
```

Comment:
I remember a step for me was working out how to intall gpg. I don't think that is a requirement for running the default build any more, so we should include instructions or a link at how to install gpg.